src/content/en/updates/2017/12/modulepreload.md

```diff
@@ -0,0 +1,138 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>Loader #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time. This article looks at how it works and why it's better for modules than &lt;link rel="preload"&gt;. #}
+
+# Preloading modules {: .page-title }
+
+{% include "web/_shared/contributors/sgomes.html" %}
+
+Browsers are finally starting to natively support ES2015 modules, both with static and
+dynamic import support. This means it's now possible to write module-based
```

Comment:
link to https://jakearchibald.com/2017/es-modules-in-browsers/ and https://developers.google.com/web/updates/2017/11/dynamic-import

Reply:
Done.